src/pages/patientView/PatientViewGenePanelModal/GenesList.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import { GenePanel, GenePanelToGene } from 'shared/api/generated/CBioPortalAPI';
+import { observer } from 'mobx-react';
+import { SimpleCopyDownloadControls } from 'shared/components/copyDownloadControls/SimpleCopyDownloadControls';
+import { serializeData } from 'shared/lib/Serializer';
+import './styles.scss';
+import { chunk } from 'lodash';
+
+interface IGenesListProps {
+    genePanelCache: GenePanel;
+    panelName: string;
+    columns?: number;
+    id?: string | undefined;
+}
+
+interface IGenesListState {
+    filter: string;
+}
+
+@observer
+export default class PatientViewGenePanelModal extends React.Component<
+    IGenesListProps,
+    IGenesListState
+> {
+    constructor(props: IGenesListProps) {
+        super(props);
+        this.state = {
```

Comment:
i think we should stick to just using mobx observables in this project (instead of traditional React component state). 
Using both will cause confusion.  So, no setState please.